Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualTaskDialog.cs

```diff
@@ -1227,7 +1227,7 @@ private void InitializeComponent()
             // 
             this._messageContent.AutoSize = false;
             this._messageContent.Font = new System.Drawing.Font("Segoe UI", 9F);
-            this._messageContent.ForeColor = System.Drawing.Color.White;
+            this._messageContent.ForeColor = System.Drawing.Color.FromArgb(((int)(((byte)(30)))), ((int)(((byte)(57)))), ((int)(((byte)(91)))));
```

Comment:
The colour should be taken from the theme

Reply:
It does, but the designer code updates when the theme is changed. Should the `ForeColor` be serialised, i.e. hidden?